lightning/src/ln/chan_utils.rs

```diff
@@ -42,8 +42,15 @@ use chain;
 
 pub(crate) const MAX_HTLCS: u16 = 483;
 
-pub(super) const HTLC_SUCCESS_TX_WEIGHT: u64 = 703;
-pub(super) const HTLC_TIMEOUT_TX_WEIGHT: u64 = 663;
+/// The fee for an HTLC-success transaction
+pub const HTLC_SUCCESS_TX_WEIGHT: u64 = 703;
+/// The fee for an HTLC-timeout transaction
+pub const HTLC_TIMEOUT_TX_WEIGHT: u64 = 663;
```

Comment:
Hmm, two notes. (a) these are used pretty extensively in channel.rs too. There's a lot of churn on the code that stuff right now, so maybe don't update it here, but need to at least open an issue for that followup, (b) why are these now pub? Do y'all need these values downstream, and should we instead expose it as an inline getter that takes `anchor` as a paratmeter (and when we do so make them module-private, not even `pub(super)`).

Reply:
Yes, we need them in `rust-lightning-signer`.

> should we instead expose it as an inline getter that takes anchor as a paratmeter (and when we do so make them module-private, not even pub(super))
This sounds great, will do.

Reply:
Note that the "make it module-private" part of the suggestion probably falls into the "mention it in a follow-up issue cause it'll conflict with five open PRs" part.

Reply:
done, I think